GearSpawner/GearSpawnReader.cs

```diff
@@ -32,7 +32,9 @@ private static string GetTrimmedLine(string line)
 		}
 		else
 		{
-			return line.Trim().ToLowerInvariant();
+			// had to remove "ToLowerInvariant" as Asset refernces are case sensitive
+			return line.Trim();
+//			return line.Trim().ToLowerInvariant();
 		}
 	}
```

Comment:
Replace the whole method with this:
`private static string GetTrimmedLine(string line) => line?.Trim() ?? "";`